src/test/java/com/saralein/server/request/RequestParserTest.java

```diff
@@ -41,20 +41,24 @@ public void setUp() {
 
     @Test
     public void returnsRequestWithCorrectInformation() {
-        Request parsedRootRequest = requestParser.parse("GET / HTTP/1.1");
+        try {
+            Request parsedRootRequest = requestParser.parse("GET / HTTP/1.1");
 
-        assertEquals(rootRequest.getMethod(), parsedRootRequest.getMethod());
-        assertEquals(rootRequest.getUri(), parsedRootRequest.getUri());
+            assertEquals(rootRequest.getMethod(), parsedRootRequest.getMethod());
+            assertEquals(rootRequest.getUri(), parsedRootRequest.getUri());
 
-        Request parsedFileRequest = requestParser.parse("GET /public/cheetara.jpg HTTP/1.1");
+            Request parsedFileRequest = requestParser.parse("GET /public/cheetara.jpg HTTP/1.1");
 
-        assertEquals(fileRequest.getMethod(), parsedFileRequest.getMethod());
-        assertEquals(fileRequest.getUri(), parsedFileRequest.getUri());
+            assertEquals(fileRequest.getMethod(), parsedFileRequest.getMethod());
+            assertEquals(fileRequest.getUri(), parsedFileRequest.getUri());
 
-        Request parsedPostRequest = requestParser.parse("POST /form HTTP/1.1\r\n\r\nBody: My=Data&More=Data");
+            Request parsedPostRequest = requestParser.parse("POST /form HTTP/1.1\r\n\r\nBody: My=Data&More=Data");
 
-        assertEquals(postRequest.getMethod(), parsedPostRequest.getMethod());
-        assertEquals(postRequest.getUri(), parsedPostRequest.getUri());
-        assertEquals(postRequest.getBody(), parsedPostRequest.getBody());
+            assertEquals(postRequest.getMethod(), parsedPostRequest.getMethod());
+            assertEquals(postRequest.getUri(), parsedPostRequest.getUri());
+            assertEquals(postRequest.getBody(), parsedPostRequest.getBody());
+        } catch (Exception e) {
+            fail("Request failed to parse in Request Parser test.");
+        }
```

Comment:
Just curious, what prompted this change?

Reply:
Adding the thrown exception from parser.